server/utils/socketsServer/socketServer.js

```diff
@@ -0,0 +1,91 @@
+const socketio = require("socket.io");
+
+const connectedUsers = {};
+let socket = null;
```

Comment:
This shouldn't be a global variable, this could cause bugs in the future. Much better to keep it's scope isolated